<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CodeCheckDialog</class>
 <widget class="QDialog" name="CodeCheckDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>420</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>模型校验结果</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="labelSummary">
     <property name="text">
      <string>Summary</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTreeWidget" name="treeResults">
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="uniformRowHeights">
      <bool>true</bool>
     </property>
     <column>
      <property name="text">
       <string>级别</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>位置</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>信息</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Close</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
